packages/manager/modules/bm-server-components/src/edit-name/template.html

```diff
@@ -0,0 +1,31 @@
+<form
+    id="bmServerEditName"
+    name="bmServerEditName"
+    data-ng-submit="bmServerEditName.$valid && $ctrl.saveName()"
+    novalidate
+>
+    <oui-modal
+        data-heading="{{:: 'server_name_edit_title' | translate }}"
+        data-primary-label="{{:: 'server_name_edit_common_save' | translate }}"
+        data-secondary-label="{{:: 'server_name_edit_common_cancel' | translate }}"
+        data-secondary-action="$ctrl.handleCancel()"
```

Comment:
```suggestion
        data-secondary-action="$ctrl.handleCancel()"
        data-primary-disabled="!bmServerEditName.$valid"
        data-primary-action="$ctrl.saveName()"
```

Reply:
saveName() is called while submitting the form. Validations are also taken care. Form will not be submitted if there is any error. Why you need to add these to oui-model?